content/advent-2015/glow-map-reduce-for-golang.md

```diff
@@ -0,0 +1,485 @@
++++
+author = ["Chris Lu"]
+date = "2015-12-27T12:11:25-08:00"
+series = ["Advent 2015"]
+title = "Glow: Map Reduce for Golang"
+
++++
+
+Having been a Java developer for many years, I simply lose interest
+in Java and want to code everything in Go, mostly due to Go's
+simplicity and performance. But it's Java who is having fun in the
+party of big data. Go is sitting alone as a wall flower. There is no
+real map reduce system for Go, until now!
+
+[Glow](https://github.com/chrislusf/glow) is aiming to be a simple
+and scalable map reduce system, all in pure Go. Not only the system
+setup is simple and scalable, but also writing and running the map
+reduce code.
+
+Glow also provides [Map()/Filter()/Reduce() functions]
+(https://github.com/chrislusf/glow/wiki#glow-apis), which works
+well in standalone mode. It's totally fine just run in standalone
```

Comment:
I'd change "It's totally fine just" to "It's totally fine to just" (adding "to")